code/__HELPERS/lists.dm

```diff
@@ -116,8 +116,7 @@
 	else
 		result = first - second
 
-	if(result.len)
-		return result
+	return result

```

Comment:
The offending lines:
https://github.com/ParadiseSS13/Paradise/blob/f80dae7fae14fc7d950492408e229ef02e5ce4b1/code/modules/mob/living/silicon/ai/ai.dm#L716-L719
https://github.com/ParadiseSS13/Paradise/blob/master/code/game/machinery/camera/camera_assembly.dm#L101-L103